packages/mdl-radio/README.md

```diff
@@ -0,0 +1,183 @@
+# MDL Radio
+
+The MDL Radio component provides a radio button adhering to the [Material Design Specification](https://material.google.com/components/selection-controls.html#selection-controls-radio-button).
+It requires no Javascript out of the box, but can be enhanced with Javascript to provide better
+interaction UX as well as a component-level API for state modification.
+
+## Installation
+
+> NOTE: Installation via NPM will be available post-alpha.
+
+## Usage
+
+```html
+<div class="mdl-radio">
+  <input class="mdl-radio__native-control" type="radio" id="radio-1"
+         name="radios" aria-labelledby="radio-1-label" checked>
```

Comment:
I don't think `aria-labelledby` is needed if you're using a label and setting its `for` property.